server/src/api/geocode/mod.rs

```diff
@@ -22,9 +23,11 @@ pub async fn geocode_address(
     let response: Vec<Point> = opencage.forward(&address).await?;
 
     if let Some(point) = response.first() {
+        let round = 10i32.pow(GEOCODE_MAX_DECIMALS) as f32;
+
         Ok(ForwardGeocodeResponse {
-            latitude: point.0.y as f32,
-            longitude: point.0.x as f32,
+            latitude: (point.0.y as f32 * round).round() / round,
+            longitude: (point.0.x as f32 * round).round() / round,
```

Comment:
We want to do the rounding here so that the rounded lat/lon is stored in the DB and there are no issues with accidentally forgetting to round on some endpoints.

Reply:
yeah makes sense